azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/AppLocalizationProvider.kt

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration
+
+internal class AppLocalizationProvider : LocalizationProvider {
+    private lateinit var language: String
+    private lateinit var customString: Map<String, String>
+
+    override fun apply(localeConfig: LocalizationConfiguration) {
+        val supportedLocales = getSupportedLanguages()
```

Comment:
is getSupportedLanguages implemented? can we uncomment if(supportedLocales.contains(localeConfig.language))?

Reply:
My bad. Pushed the draft too soon. Thanks :)